src/CryptoAxus.API/Controllers/ArtistController.cs

```diff
@@ -262,6 +305,20 @@ private IReadOnlyList<Links> CreateArtistLinks(string? id = null, int? userId =
                                        $"{HttpContext?.Request.Scheme}://{HttpContext?.Request.Host}{Constants.ApiValue}");
         links.Add(link);
 
+        link = new Links(Url.RouteUrl("GetOffersReceivedByArtist", new { userId }),
+                         "get_offersRecivedByArtist",
+                         Constants.GetMethod);
+        link.Href = link.Href?.Replace(Constants.ApiValue,
+                                       $"{HttpContext?.Request.Scheme}://{HttpContext?.Request.Host}{Constants.ApiValue}");
+        links.Add(link);
+        return links.AsReadOnly();
+    }
+
+    private IReadOnlyList<Links> CreateOfferLinks(OffersDto dto, string? field)
+    {
+        Links link;
+        List<Links> links = new List<Links>();
+
         return links.AsReadOnly();
     }
```

Comment:
Wrong implementation. This is just returning an empty list of links to the client end.

Reply:
Resolved